src/libraries/System.Text.Json/Common/JsonNumberHandling.cs

```diff
@@ -29,6 +29,9 @@ enum JsonNumberHandling
 
         /// <summary>
         /// Numbers will be written as JSON strings (with quotes), not as JSON numbers.
+        /// <remarks>
```

Comment:
Just seeing the comment in the PR in the API docs repro https://github.com/dotnet/dotnet-api-docs/pull/7488#issuecomment-998142017. Let's move this into the summary section, here, below, and in that PR. Sorry for the back and forth.

Reply:
All right. No problem, have now adjusted this.